Lib/multiprocessing/context.py

```diff
@@ -256,6 +256,8 @@ def get_start_method(self, allow_none=False):
     def get_all_start_methods(self):
         if sys.platform == 'win32':
             return ['spawn']
+        elif sys.platform == 'darwin':
+            return ['spawn', 'fork', 'forkserver']
```

Comment:
Does this not require the `if reduction.HAVE_SEND_HANDLE:` check as below?

Reply:
Moved the platform check to be included in the HAVE_SEND_HANDLE if block